### PR TITLE
Add docs for running bot

### PR DESCRIPTION
**Pull request by Newlifer**

### CODE REVIEW

**Comment by jt3k** [comment]
grand mersi

File: README.md [hunk 1]
```diff
@@ -23,7 +23,11 @@ To determine Telegram room parameters, follow the procedure:
    ```
 5. To make all room messages visible to the bot, make it a room admin
 
-Run `node app.js` to start the bot.
+To start the bot:
+
++ Set NODE_ENV environment variable to value that is in your application configuration. For example, in [sample-config.json]( https://github.com/Newlifer/chat-linker/blob/master/sample-config.json) there're two: `prod` for production sessings and `dev` for development.
```
Comment (ForNeVeR) [inline README.md]: Please update this URL to be relative, e.g. `(sample-config.json)`. It shouldn't include the absolute repo path.

Comment (Newlifer) [inline README.md]: Thanks! Done: https://github.com/jt3k/chat-linker/pull/21

Comment (jt3k) [inline README.md]: heh   to start the bot   i use  `npm run dev`  or `npm run prod`.  now i fixed this text in repo

thnx!

**Pull request merged**